app-creator/src/redux/reducer.ts

```diff
@@ -89,6 +92,43 @@ export const reducer: Reducer<AppCreatorStore, AppCreatorAction | AnyAction> = (
         ...state,
         eventType: action.payload.value ? EventType.reordering : EventType.none,
       };
+    case ADD_WIDGET_META_DATA:
+      return {
+        ...state,
+        template: {
+          ...state.template,
+          ...action.payload,
+        },
+      };
+    case UPDATE_WIDGET_META_DATA:
+      const updatedTemplate = { ...state.template };
+      updatedTemplate[action.payload.id][action.payload.attributeType][
+        action.payload.key
+      ] = action.payload.value;
+
+      const {
+        widgetRef,
+        style: { borderWidth, borderColor, borderType },
+      } = updatedTemplate[action.payload.id];
+
+      updatedTemplate[
+        action.payload.id
+      ].style.border = `${borderWidth} ${borderType} ${borderColor}`;
```

Comment:
Were the border styles not applied without expliciting setting the border attribute this way?

Reply:
Removed this, they work separately. Just had to change borderType to borderStyle.